dev/com.ibm.ws.javaee.ddmodel_fat/bnd.bnd

```diff
@@ -23,6 +23,10 @@ src: \
 
 test.project: true
 
+# Not sure why these in particular must be added.
```

Comment:
I would remove the comment.  If you don't want them you can add connectors-2.1 to the server.xml for the Jakarta EE 10 server xml file.

Reply:
I've updated the comment and refreshed the PR.